base_layer/core/src/blocks/blockheader.rs

```diff
@@ -160,6 +160,10 @@ impl BlockHeader {
         prev_pow.add_difficulty(&self.pow, self.achieved_difficulty());
         prev_pow.total_accumulated_difficulty()
     }
+
+    pub fn into_builder(self) -> BlockBuilder {
+        BlockBuilder::new(self.version)
```

Comment:
```suggestion
        BlockBuilder::new(self.version).with_header(self)
```
This should fix the tests

Reply:
Lol. My bad